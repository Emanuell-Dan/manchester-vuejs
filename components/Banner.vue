<template>
  <div class="banner" />
</template>

<script>
export default {
	name: 'Banner'
};
</script>

<style lang="scss">
.banner {
	background-image: url('../static/manchester-banner.png');
	background-position: center;
	background-repeat: no-repeat;
	background-size: cover;
	height: 100%;
}
</style>
