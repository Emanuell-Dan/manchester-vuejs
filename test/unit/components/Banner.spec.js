import Banner from '@/components/Banner.vue';
import { mount } from 'vue-test-utils';

describe('Banner.vue', () => {
	let cmp;

	beforeEach(() => {
		cmp = mount(Banner);
	});

	it('should render with expected html structure', () => {
		expect(cmp.element).toMatchSnapshot();
	});
});
