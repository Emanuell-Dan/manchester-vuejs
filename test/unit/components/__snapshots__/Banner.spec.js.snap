// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner.vue should render with expected html structure 1`] = `
<div
  class="banner"
/>
`;
